scenarios/banda_1852/prior.py

```diff
@@ -79,12 +80,13 @@ def rvs(self):
         delta_logw = self.delta_logw.rvs()
         depth_offset = self.depth_offset.rvs()
         params = np.array(latlon+[mag,delta_logl,delta_logw,depth_offset])
-        return pd.Series(params,["latitude",
-                                 "longitude",
-                                 "magnitude",
-                                 "delta_logl",
-                                 "delta_logw",
-                                 "depth_offset"])
+        random_variates = pd.Series(params,["latitude",
+                                            "longitude",
+                                            "magnitude",
+                                            "delta_logl",
+                                            "delta_logw",
+                                            "depth_offset"])
+        return random_variates
```

Comment:
Just making the code a little more readable here, returning a variable instead of a long pd.Series command